asteroid_filterbanks/enc_dec.py

```diff
@@ -265,13 +274,14 @@ def forward(self, spec) -> torch.Tensor:
         """
         filters = self.get_filters()
         spec = self.filterbank.pre_synthesis(spec)
-        return multishape_conv_transpose1d(
+        wav = multishape_conv_transpose1d(
```

Comment:
We need to pass a `kwargs` to `post_synthesis` in order to support the `length` argument. But it seems unfair to pass it only to `post_synthesis`.
Three ugly ways to do: 
- Pass all kwargs to each method `pre_synthesis`, `post_synthesis` (need `**kwargs` in the child hooks to accept unknown arguments).
- Pass `pre_kwargs` and `post_kwargs` dict to `forward` (super ugly on the user side)
- Receive all `**kwargs`, analyze the signature of both functions and pass accordingly. This looks like the most correct way to do. It's not too hard to implement but I wonder if it's worth it. 
Thoughts @jonashaag

Reply:
Shouldn't length be part of the decoder? It's useful for other filter banks as well, right? And I think the trimming/padding logic is the same for other filter banks (trim or zero pad [centered or not centered]).

Reply:
I just thought about this possibility as well, my bad. 
So yes, this will be much better.